client/rest-high-level/src/main/java/org/elasticsearch/client/RestHighLevelClient.java

```diff
@@ -112,6 +113,14 @@ public boolean ping(Header... headers) throws IOException {
     }
 
     /**
+     * Get the cluster info otherwise provided when calling GET `localhost:9200`
+     */
+    public MainResponse info(Header... headers) throws IOException {
```

Comment:
I only found https://www.elastic.co/guide/en/elasticsearch/reference/master/zip-targz.html#_checking_that_elasticsearch_is_running, would that be something you want me to link to?